ngraph_bridge/ngraph_builder.cc

```diff
@@ -5153,12 +5141,39 @@ Status Builder::TranslateGraph(
     return n->is_parameter() || is_result;
   };
 
+  size_t num_tags = 0;
   for (auto n : ng_function->get_ordered_ops()) {
     // Results and Parameters are not expected to have provenance tags
     if (!check_if_result_or_parameter(n)) {
-      if (n->get_provenance_tags().size() == 0) {
-        return errors::Internal("Found ngraph node ", n->get_name(),
-                                " which does not have provenance tag set");
+      num_tags = n->get_provenance_tags().size();
```

Comment:
We'll have change this check once prov tags are added for Parameters as well.

Reply:
yes